lib/style-reader.js

```diff
@@ -159,6 +159,8 @@ function documentMatcherRule() {
             // TODO: handle unknown document matchers
         }
     });
+
+    var smallCaps = identifierToConstant("smallCaps", documentMatchers.smallCaps);
```

Comment:
For consistency with the other style mappings, this should be `small-caps` rather than `smallCaps`.